appengine/accessible/workspaceview.component.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Blockly Games: Accessible
+ *
+ * Copyright 2016 Google Inc.
+ * https://github.com/google/blockly-games
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how a Blockly.Workspace is
+ * rendered in AccessibleBlockly. Modified to run music code.
```

Comment:
I should have added a comment here. I 100% agree with what you said, Sean. I originally tried to put the 'Run Code' button out in the demo and have the programmer from the demo be able to inject it into a specific spot in the accessibleBlockly app. However, that caused huge errors when it came to tab navigation. After talking it over with Neil, we decided to keep as is so that we have something working with the understanding that this issue should be looked into more carefully later. If it's all right with you, I'll add a TODO at the runCode() line to say that once that's resolved, this file can be removed altogether?